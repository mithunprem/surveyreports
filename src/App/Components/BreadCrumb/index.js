import React, { Fragment } from 'react';
import PropTypes from 'prop-types';
import { Breadcrumb as Crumb, BreadcrumbItem } from 'reactstrap';

const Breadcrumb = ({ pageName }) => {
  return (
    <Fragment>
      <Crumb tag="nav" listTag="div">
        <BreadcrumbItem tag="a" href="/">Home</BreadcrumbItem>
        <BreadcrumbItem active tag="span">{pageName}</BreadcrumbItem>
      </Crumb>
    </Fragment>
  );
}

Breadcrumb.propTypes = {
  pageName: PropTypes.string
}

export default Breadcrumb;
