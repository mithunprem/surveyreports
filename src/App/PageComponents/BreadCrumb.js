import React, { Fragment } from 'react';
import { Breadcrumb as Crumb, BreadcrumbItem } from 'reactstrap';

const Breadcrumb = ({ name }) => {
  return (
    <Fragment>
      <Crumb tag="nav" listTag="div">
        <BreadcrumbItem tag="a" href="/">Home</BreadcrumbItem>
        <BreadcrumbItem active tag="span">{name}</BreadcrumbItem>
      </Crumb>
    </Fragment>
  );
}

export default Breadcrumb;
