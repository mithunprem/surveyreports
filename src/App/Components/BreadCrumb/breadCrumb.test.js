import React from 'react';
import { create } from 'react-test-renderer';
import BreadCrumb from '../../Components/BreadCrumb';

describe("BreadCrumb", () => {
  test("should render without throwing an error", () => {
    const breadCrumb = create(<BreadCrumb pageName="test" />).toJSON();
    expect(breadCrumb).toMatchSnapshot();
  });
});
