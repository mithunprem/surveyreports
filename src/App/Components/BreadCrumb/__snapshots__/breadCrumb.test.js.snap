// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreadCrumb should render without throwing an error 1`] = `
<nav
  aria-label="breadcrumb"
  className=""
>
  <div
    className="breadcrumb"
  >
    <a
      className="breadcrumb-item"
      href="/"
    >
      Home
    </a>
    <span
      aria-current="page"
      className="active breadcrumb-item"
    >
      test
    </span>
  </div>
</nav>
`;
